app/uk/gov/nationalarchives/omega/editorial/connectors/ApiConnector.scala

```diff
@@ -120,6 +130,8 @@ object ApiConnector {
     case object GetLegalStatuses extends SID("OSLISALS001")
     // TODO: The real SID will be provided by Adam once he figures out the schema.
     case object GetPlacesOfDeposit extends SID("OSGPOD001")
+    case object GetPersons extends SID("OSGPER001")
```

Comment:
Are these SIDs real, or stopgaps? 

I don't see any reference to them in the ticket.

Reply:
These SID are made up, I can't find any real SIDs for these.

Reply:
That's what I thought. In that case, maybe copy the TODO above, to each? Or something like: the following SIDs are made up, but will be replaced ...